ui/app.py

```diff
@@ -18,15 +18,35 @@
 limitations under the License.
 """
 
-from flask import Flask, render_template
+from flask import Flask, render_template, send_from_directory
+
+from config.flask import secret_key, debug
 
 app = Flask(__name__)
+app.secret_key = secret_key
+
+
+# Prevent cached response when running in debug mode, so we can easily
+# rebuild on change and see the differences.
+if debug:
+    @app.after_request
+    def after_request(response):
+        response.headers["Cache-Control"] = "no-cache, no-store, must-revalidate, public, max-age=0"
+        response.headers["Expires"] = 0
+        response.headers["Pragma"] = "no-cache"
+        return response
+
+
+@app.route('/<path:path>', methods=['GET'])
+def frontend_proxy(path):
+    """Serves the unbound paths from the Angular compiled directory."""
+    return send_from_directory('./web/dist', path)
 
 
-@app.route('/', methods=['GET'])
+@app.route('/')
```

Comment:
Nit: any reason not to specify GET here (probably the default) but specifying it in the previous handler? I'd advocate for consistency, either remove both or specify both.

Reply:
> By default a rule just listens for GET (and implicitly HEAD).

https://flask.palletsprojects.com/en/1.1.x/api/#flask.Flask.route


I removed the methods.